cypress/integration/login.spec.js

```diff
@@ -0,0 +1,64 @@
+describe('Test For Login Page and Forgot Password', function() {
+	context('Login Page', function() {
+		beforeEach(function() {
+			cy.visit('localhost:3000/login');
+        })
+        it('test login no input fileds',function(){
+            cy.get('.button.is-success').click()
+            cy.url().should('include', '/')
```

Comment:
we should test that it is still on login page , `cy.url().should('include', '/login')`